src/plugins/intel_cpu/src/nodes/rnn.cpp

```diff
@@ -292,6 +292,7 @@ RNN::RNN(const std::shared_ptr<ov::Node>& op, const dnnl::engine& eng, WeightsSh
     S = statesCount(cell_type);
     SC = rnnCellBase->get_hidden_size();
     N = {getInputShapeAtPort(0).getMinDims()[0], getInputShapeAtPort(0).getMaxDims()[0]};
+    P = (cell_type == dnnl::algorithm::vanilla_lstm && op->get_input_size() == 7) ? 3 : 0;
```

Comment:
What do mean these magic numbers `3` and `0` ?
Could you add constants with descriptive name ?

Reply:
I just removed these constants here and obtain real numbers directly from the operation.